dist/index.html

```diff
@@ -7,6 +7,166 @@
   <body>
     <section class="all-recipes">
       <h1> Our recipes </h1>
+      <section class = "tag-area">
+        <div class="tag-rows">
+          <div class="tag-row row-one">
+            <div class = "tag-card">
+              <div class="tag-image-bg active-bg">
+                <img class = "tag-image" src = "./images/breakfast.png">
+              </div>
+              <div class="tag-option">
+                <p class="tag-text">Appetizer</p>
+              </div>
```

Comment:
I see that `tag-image-bg` has different styling than `tag-image` so that div is necessary, but there is a div around the text as well with the only styling being text align center. Could we do that styling directly on the class of `tag-text` and remove the surrounding div? Any flexbox will still work since the p tag will just be the sibling of the `tag-image-bg` div. 

Let me know what you think or if there is a reason for the div that I am missing.

Reply:
It was made when I was trying to create the sub options. I agree, it's not too useful. Have made the change.